node/network/collator-protocol/src/collator_side/mod.rs

```diff
@@ -75,6 +82,7 @@ impl Metrics {
 struct MetricsInner {
 	advertisements_made: prometheus::Counter<prometheus::U64>,
 	collations_sent: prometheus::Counter<prometheus::U64>,
+	collations_sent_requested: prometheus::Counter<prometheus::U64>,
```

Comment:
Nit: I think grammatically it would be more correct to use "send". In this case "requested" gets the 3rd participle form.